lib/crawler/accounts/schemas/user.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Crawler.Accounts.Schemas.User do
```

Comment:
To follow our convention https://nimblehq.co/compass/development/code-conventions/elixir/phoenix/#business-domain

```suggestion
defmodule Crawler.Account.Schemas.User do
```

Do not forget to change the folder name in `lib/crawler/accounts/schemas/user.ex` -> `lib/crawler/account/schemas/user.ex`

Reply:
Thanks! Fixed in [c5ceeda](https://github.com/vnntsu/elixir-ic/pull/31/commits/c5ceeda8e3ae39f2b09d5e017137cd44a5a8aded)